### DISCUSSION TITLE
Control alert chrome after adding extension chrome

### DISCUSSION THREAD

**Discussion by Noormisa** [getgauge/taiko#1963] [Development]
Can anyone help me? I want to do an automatic test for adding a chrome extension with taiko, it's possible

**Comment by sriv** [comment]
If I understand correctly, you'd like to install an extension in chromium using taiko?

If so, this is not possible, since taiko works only within the page and cannot control other parts of the browser.

However, you can launch chromium with the `--load-extension <comma separated path list to extensions>` flag and chromium should launch with the extensions added.
